libse/Utilities.cs

```diff
@@ -852,8 +852,25 @@ public static void ShowHelp(string parameter)
             {
                 helpFile = "https://www.nikse.dk/SubtitleEdit/Help";
             }
-
-            System.Diagnostics.Process.Start(helpFile + parameter);
+            try

```

Comment:
Could this be just `UiUtil.OpenURL(helpFile + parameter)` ?

Looks good :)
Nice to encapsulate "process start" as it also seem broken in dot net core: https://github.com/dotnet/runtime/issues/17938

Reply:
This file is in libse, and I didn't want to create a dependency on SubtitleEdit for this.